### ISSUE TITLE
It pauses and blurs video when I want to see the wallpaper from bottom right corner.

### ISSUE THREAD

**Issue by laichiaheng** [luisbocanegra/plasma-smart-video-wallpaper-reborn#37]
https://github.com/user-attachments/assets/b075a78e-aa3d-4f0d-9191-ec00c8bf22e8

**Comment by luisbocanegra** [comment]
Is this with the same configuration as https://github.com/luisbocanegra/plasma-smart-video-wallpaper-reborn/issues/35? If not please send the updated one.

The only configuration I am able to reproduce this with is

- **Pausing video** > **Maximized or full-screen windows** selected and showing the desktop (the action you are triggering with the corner). It seems that either https://github.com/KDE/plasma-workspace/tree/master/libtaskmanager does not detect when it's showing the desktop unless pause video is set to **Active window is present**

If the above matches your configuration there is a workadound you can use, in the **Desktop Effects** section set **Play in** to `windowaperture`

![Screenshot_20240714_224050](https://github.com/user-attachments/assets/94e8bc94-1158-4d4d-9bed-1e855d397902)

**Comment by laichiaheng** [comment]
> Is this with the same configuration as #35? If not please send the updated one.
> 
> The only configuration I am able to reproduce this with is
> 
>     * **Pausing video** > **Maximized or full-screen windows** selected and showing the desktop (the action you are triggering with the corner). It seems that either https://github.com/KDE/plasma-workspace/tree/master/libtaskmanager does not detect when it's showing the desktop unless pause video is set to **Active window is present** or some setting I missed
> 
> 
> If the above matches your configuration there is a workadound you can use, in the **Desktop Effects** section set **Play in** to `windowaperture`
> 
> ![Screenshot_20240714_224050](https://private-user-images.githubusercontent.com/15076387/348600805-94e8bc94-1158-4d4d-9bed-1e855d397902.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEwMTk0NjksIm5iZiI6MTcyMTAxOTE2OSwicGF0aCI6Ii8xNTA3NjM4Ny8zNDg2MDA4MDUtOTRlOGJjOTQtMTE1OC00ZDRkLTliZWQtMWU4NTVkMzk3OTAyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE1VDA0NTI0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNlYzQ4NDgzYzllNmVmN2NlODc1Mzk2YWIwYzVjOWYwNDdlNGUzYWIxOTE0ODc2ZDczNGQxYjdiN2E1NWIxZGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.M4dvaBOCA2gtCJ_lRbcgQL4rYF9VA2x13jf6LecuDCQ)

Nope, I enabled lock screen detection, but it was not fullscreen.

**Comment by luisbocanegra** [comment]
What happens if you put `windowaperture` as per the screenshot? Is the pausing/resuming working correctly when **pause video** is set to **Active windows is present** (Filter option should be checked) and you click on the desktop?

I am having trouble reproducing the problem here. ¿Does setting all exactly like this makes it resume when the desktop is shown?

![Screenshot_20240714_232128](https://github.com/user-attachments/assets/ffd65549-e2ef-4677-9781-388fc4b6f986)

**Comment by laichiaheng** [comment]
> What happens if you put `windowaperture` as per the screenshot? Is the pausing/resuming working correctly when **pause video** is set to **Active windows is present** (Filter option should be checked) and you click on the desktop?
> 
> I am having trouble reproducing the problem here. ¿Does setting all exactly like this makes it resume when the desktop is shown?
> 
> ![Screenshot_20240714_232128](https://private-user-images.githubusercontent.com/15076387/348605111-ffd65549-e2ef-4677-9781-388fc4b6f986.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEwMjkwNzgsIm5iZiI6MTcyMTAyODc3OCwicGF0aCI6Ii8xNTA3NjM4Ny8zNDg2MDUxMTEtZmZkNjU1NDktZTJlZi00Njc3LTk3ODEtMzg4ZmM0YjZmOTg2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE1VDA3MzI1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVkZjgyYjQyYmZiZDI4NzRhNWY0MTljNzc5YWIxYTgyOGQwNGNlOGI1N2I0ZTJhZmU4MWQ5NWJkYTZlZDAwOTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.q2zgclMP8pVdU35uW6Cv6DlzWS0sx3SXVup6HZFBHFA)

Yes, it works as expected.

**Comment by luisbocanegra** [comment]
> Maybe it could become the default setting?

Probably, yeah.

But first I want to improve the way effects are configured, probably giving each effect a friendly name and make them easily select-able or something.
